indexer/src/test/java/uk/ac/ebi/uniprot/indexer/uniprotkb/processor/UniProtEntryConverterTest.java

```diff
@@ -66,12 +68,28 @@
 
     private TaxonomyRepo repoMock;
     private GoRelationRepo goRelationRepoMock;
+    private HashMap<String, SuggestDocument> suggestions;
+
+//    //    @Test
```

Comment:
is it possible to remove these commented line?

Reply:
yes!